src/App.js

```diff
@@ -80,20 +80,6 @@ export default class App extends Component {
 
     return (
       <div className="App">
-        <header className="App-header" role="banner">
-          <div className="app-title">
-            <h1 className="maroon app-title"> {title} </h1>  
-          </div>
-          <span className="about-icon">
-            <OverlayTrigger placement="bottom" overlay={popover}>
-              <FontAwesomeIcon 
```

Comment:
Was it discussed that these changes will be removing the About popup? I know we wanted to remove the top banner, but if the About button was still desired it should be moved to another spot.